src/Module/Api/Friendica/Statuses/DislikedBy.php

```diff
@@ -41,12 +41,12 @@ protected function rawContent(array $request = [])
 		$uid = self::getCurrentUserID();
 
 		if (empty($this->parameters['id'])) {
-			DI::mstdnError()->UnprocessableEntity();
+			$this->logAndJsonError(422, $this->errorFactory->UnprocessableEntity());
```

Comment:
Can't we create some constants for the HTTP error numbers?

Reply:
See https://github.com/friendica/friendica/pull/13530#discussion_r1359480492